tests/test_frontend_type_conversion.py

```diff
@@ -1,11 +1,11 @@
 from __future__ import annotations
 
 import ast
-from sys import _getframe
+from sys import _getframe  # type: ignore
 from typing import Any, Callable, Generic, Literal, Tuple, TypeAlias, TypeVar
 import pytest
 
-from xdsl.frontend.dialects.builtin import _FrontendType
+from xdsl.frontend.dialects.builtin import _FrontendType  # type: ignore
```

Comment:
Why do you need a type: ignore here?

Reply:
underscore means private, it doesn't like that